test/functional/application_controller_subclass_test.rb

```diff
@@ -135,20 +135,6 @@ class TestableResourcesControllerTest < ActionController::TestCase
     end
   end
 
-  context "secure headers in HTTP response" do
```

Comment:
secure headers are now added via middleware, so are not available to functional tests.